### ISSUE TITLE
Command Line Options

### ISSUE THREAD

**Issue by teddue** [wooxo/Vhd2Disk#3]
Are there any command line Options for Vhd2Disk?

If not, do you plan to add those in a future release?

**Comment by orkblutt** [comment]
Hi,

no it doesn't take any command line option. I do not have any plan to work on this old tool.
Let me know what command lines you need. I'll think about it.

cheers,

orkblutt

**Comment by teddue** [comment]
I would like a command line option for choosing the VHD I intend to apply. 
Something like vhd2disk.exe /path <path to file>

**Comment by Zero3K** [comment]
I agree with that suggestion. It should be along the lines of vhd2disk D:\ramdisk.vhd R: (replacing the file path and drive to restore to with whatever one is being chosen by the user).
